src/erasure-code/isa/ErasureCodeIsaTableCache.cc

```diff
@@ -62,7 +62,7 @@ ErasureCodeIsaTableCache::~ErasureCodeIsaTableCache()
       for (table_it = tables_it->second.begin(); table_it != tables_it->second.end(); ++table_it) {
         if (table_it->second) {
           if (*(table_it->second)) {
-            delete *(table_it->second);
+            delete[] *(table_it->second);
```

Comment:
Memory of `table_it->second` may be allocated by `malloc` in the c file, but `new` cannot be used in the c file.

Reply:
@Svelar i don't think it's true. could you help me understand how come `ErasureCodeIsa` calls into `ErasureCodeJerasure`?

Reply:
Sorry for my mistake. Should point to src/erasure-code/shec/ErasureCodeShecTableCache.cc.
Ignore plz~

Reply:
anything else you request to change? otherwise could you please dismiss your request to change?

Reply:
Looks good, dismiss done.